MatrixSDK/NotificationCenter/MXNotificationCenter.h

```diff
@@ -208,12 +209,14 @@ extern NSString *const kMXNotificationCenterAllOtherRoomMessagesRuleID;
  @param notify enable/disable notification.
  @param soundName the name of the sound to apply (`ring`, `default`, etc).
  @param highlight enable/disable highlight option.
+ @param completion an optional completion block for the operation.
  */
 - (void)updatePushRuleActions:(NSString*)ruleId
                          kind:(MXPushRuleKind)kind
                        notify:(BOOL)notify
                     soundName:(NSString*)soundName
-                    highlight:(BOOL)highlight;
+                    highlight:(BOOL)highlight
+                   completion: (void (^_Nullable)(NSError * _Nullable error))completion;
```

Comment:
I personally find the `completion: (nullable void (^)(NSError * _Nullable error))completion;` version easier to prse

Reply:
Yeah, it looks better tbh!
https://github.com/matrix-org/matrix-ios-sdk/pull/1702/commits/1f5997c8aee6cc36bccbd1b3ac1cbc37c020aca2